### PR TITLE
ECHO-592 Change border corder of browse image container.

### PR DESCRIPTION
**Pull request by mightynimble**
browse-scaler changes are on branch ```feature/echo-592``` in ```echo-browser-scaler``` which is behind firewall (not in stash), so no pull request for that :(

Well, the best way to review this is through a gotomeeting so that you don't have to set up the ECHO stack.

### CODE REVIEW

**Comment by bilts** [comment]
Can you explain why we're changing the border color? I think the design team set the current border color.

**Comment by mightynimble** [comment]
I will, when we are on the goto meeting later.

**Comment by mightynimble** [comment]
I will send you the browse-scaler controller changes in an email.

File: spec/features/collections/collection_results_spec.rb [hunk 1]
```diff
@@ -55,13 +55,6 @@
     expect(page).to have_css('.badge-opendap')
   end
 
-  it "displays a placeholder for collections which have no thumbnail URLs" do
```
Comment (bilts) [inline spec/features/collections/collection_results_spec.rb]: This spec should still be here, but just assert the image is image_unavailable.svg

**Comment by bilts** [comment]
Small fix, but otherwise I approve of this pull request, including the scaler changes you sent via email.

Please merge/deploy the browse scaler changes before merging this.

**Pull request merged**